### DISCUSSION TITLE
Erroneous comment about need to address AIRFLOW-1455 in airflow_local_settings.py

### DISCUSSION THREAD

**Discussion by vitaly-krugl** [apache/airflow#24831] [General]
### Apache Airflow version

2.3.2 (latest released)

### What happened

The latest airflow_local_settings.py contains the following out-of-date comment concerning the need to fix AIRFLOW-1455...
(see https://github.com/apache/airflow/blob/7de050ceeb381fb7959b65acd7008e85b430c46f/airflow/config_templates/airflow_local_settings.py#L28-L31)

```
# TODO: Logging format and level should be configured
# in this file instead of from airflow.cfg. Currently
# there are other log format and level configurations in
# settings.py and cli.py. Please see AIRFLOW-1455.
LOG_LEVEL: str = conf.get_mandatory_value('logging', 'LOGGING_LEVEL').upper()
```

However, according to https://lists.apache.org/thread/zfw6xbb4846n8l80xsky3joppghkjd77, this issue was resolved as "Won't Fix".

Please remove the erroneous comment from airflow_local_settings.py.

### What you think should happen instead

Erroneous comment about AIRFLOW-1455 should be removed from airflow_local_settings.py because according to https://lists.apache.org/thread/zfw6xbb4846n8l80xsky3joppghkjd77, this issue was resolved as "Won't Fix".

### How to reproduce

Open airflow/config_templates/airflow_local_settings.py and find the erroneous comment:

```
# TODO: Logging format and level should be configured
# in this file instead of from airflow.cfg. Currently
# there are other log format and level configurations in
# settings.py and cli.py. Please see AIRFLOW-1455.
```

### Operating System

OSX and Linux

### Versions of Apache Airflow Providers

N/A

### Deployment

Docker-Compose

### Deployment details

OSX and Linux, but N/A for the purposes of this issue.

### Anything else

N/A

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Can you please remove it? It should be easy.

**Comment by potiuk** [comment]
Airlfow is created by > 2100 contributors. It's far easier to remve the comment in a PR than open an issue about it.

**Reply by squerez** [reply]
Has this issue been addressed? I can still see this comment on `airflow/config_templates/airflow_local_settings.py`

**Reply by potiuk** [reply]
@joao-fm-santos -> Airlfow is created by > 2200 contributors (number grows). It's far easier to remve the comment in a PR than open an issue about it.
